http-transport-binding.md

```diff
@@ -103,33 +103,33 @@ identically to [HTTP over TLS]([RFC2818][RFC2818]).
 This specification does not further define any of the [CloudEvents][ce] event
 attributes.
 
-Two of the event attributes, `datacontenttype` and `data` are handled specially
-and mapped onto HTTP constructs, all other attributes are transferred as
-metadata without further interpretation.
+The `datacontenttype` attribute is handled specially and mapped onto HTTP
+constructs, all other attributes are transferred as metadata without further
+interpretation.
```

Comment:
I think it's still worth mentioning that the data payload is mapped onto HTTP constructs.

Maybe:

> The data payload and the `datacontenttype` attribute is handled specially and mapped onto HTTP constructs, all other attributes are transferred as metadata without further interpretation.

❓